packages/analytics/analytics-utilities/src/queryTime.spec.tz.ts

```diff
@@ -444,7 +439,7 @@ runDstTest('daylight savings time: spring', () => {
   })
 })
 
-runDstTest('daylight savings time: fall', () => {
+;(supportsDst ? describe : describe.skip)('daylight savings time: fall', () => {
```

Comment:
@adamdehaven is this an acceptable pattern to use here?  Trying to inline the check to get past the previous run's build errors:

https://github.com/Kong/public-ui-components/actions/runs/4768426700